tests/unittest_brain_pathlib.py

```diff
@@ -0,0 +1,42 @@
+import pytest
```

Comment:
Just for "totality" could you also add a test for the inference of `parents` itself?

Reply:
I feel like I'm almost there, but it's not currently working for:

```py
from pathlib import Path


def _main() -> None:
    current_path = Path().resolve()
    path_parents = current_path.parents
    parent_path = path_parents[2].resolve()
    print(f"{current_path=}")
    print(f"{parent_path=}")


if __name__ == "__main__":
    _main()
```

It fails with the same `test.py:7:18: E1101: Instance of 'tuple' has no 'resolve' member (no-member)`.

Seems to be why my attempt at adding a nicer-looking test failed:

```py
def test_inference_parents_subscript_index() -> None:
    """Test inference of ``pathlib.Path.parents``, accessed by index."""
    parents, path = astroid.extract_node(
        """
    from pathlib import Path

    current_path = Path().resolve()
    path_parents = current_path.parents
    path_parents  #@
    path_parents[2]  #@
    """
    )
    inferred = parents.inferred()
    assert len(inferred) == 1
    assert isinstance(inferred[0], bases.Instance)
    assert inferred[0].qname() == "pathlib._PathParents"

    inferred = path.inferred()
    assert len(inferred) == 1
    if PY310_PLUS:
        assert isinstance(inferred[0], bases.Instance)
        assert inferred[0].qname() == "pathlib.Path"
    else:
        assert inferred[0] is Uninferable
```